### ISSUE TITLE
🚀 Feature Request: Hook for set CSS Print Method as Internal Embedding

### ISSUE THREAD

**Issue by sidiraihan** [elementor/elementor#17841]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

as developer i have wordpress multisite with wp /upload folder always replaced to empty because of containerization so my elementor css always reset to empty to solve this problem i want hook to set elementor CSS Print Method as Internal Embedding via functions.php

### Describe the solution you'd like

it would be nice if this CSS Print Method hook exist

### Additional context

![image](https://user-images.githubusercontent.com/21240420/153999467-074c5809-a321-45c5-8983-51e18d748ee9.png)

**Comment by rami-elementor** [comment]
@sidiraihan You want to override user selection with this hook?

**Comment by sidiraihan** [comment]
> @sidiraihan You want to override user selection with this hook?

Yes @rami-elementor because of WordPress multisite it's hard to change 1 by 1 per site from elementor settings
